web/src/main/java/com/softserve/osbb/config/Application.java

```diff
@@ -1,25 +1,40 @@
+/*
+ * Project “OSBB” – a web-application which is a godsend for condominium head, managers and 
+ * residents. It offers a very easy way to manage accounting and residents, events and 
+ * organizational issues. It represents a simple design and great functionality that is needed 
+ * for managing. 
+ */
 package com.softserve.osbb.config;
 
-import org.springframework.boot.autoconfigure.EnableAutoConfiguration;
+import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.boot.builder.SpringApplicationBuilder;
 import org.springframework.boot.context.web.SpringBootServletInitializer;
-import org.springframework.context.annotation.ComponentScan;
-import org.springframework.context.annotation.Configuration;
 
 /**
+ * Initializes this Spring Boot application.
+ * 
  * Created by nataliia on 24.07.16.
+ * @version 1.1
+ * @since 15.11.2016 
  */
-
-@Configuration
-@EnableAutoConfiguration
-@SpringBootApplication
-@ComponentScan(basePackages = {"com.softserve.osbb"})
+@SpringBootApplication(scanBasePackages = "com.softserve.osbb")
 public class Application extends SpringBootServletInitializer {
 
+    /**
+     * Bootstraps Spring Boot application.
+     * @param args CLI arguments.
+     */
+    public static void main(String[] args) {
+        SpringApplication.run(Application.class, args);
+    }
+    
+    /**
+     * {@inheritDoc}}
```

Comment:
Doesn't @inheritdoc tag indicates that an element should inherit its documentation from its parent class?